assets/javascripts/discourse/widgets/discourse-reactions-counter.js.es6

```diff
@@ -74,7 +139,8 @@ export default createWidget("discourse-reactions-counter", {
         this.attach(
           "discourse-reactions-state-panel",
           Object.assign({}, attrs, {
-            statePanelExpanded: this.state.statePanelExpanded
+            statePanelExpanded: this.state.statePanelExpanded,
+            state: this.state
```

Comment:
not really liking this, sharing state like sounds like a smell and might lead to zombie objects... We can keep it for now as I understand the hard problem it solves. But we need something better.

Reply:
I know it's weird but I didn't found any better way to do this😅